.github/workflows/build_and_run_selected_test.yml

```diff
@@ -0,0 +1,117 @@
+name: Build and Run Selected Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      test-selection:
+        description: 'Specification of selected test(s), as accepted by pytest -k'
+        required: true
+        type: string
+      run-count:
+        description: 'Number of test runs to perform'
+        required: true
+        type: number
+      archs:
+        description: 'Archs to run tests on, e. g.: x64, arm64'
+        default: 'x64, arm64'
+        required: true
+        type: string
+      build_types:
+        description: 'Build types to run tests on, e. g.: debug, release'
+        default: 'debug'
+        required: true
+        type: string
+      pg-version:
+        description: 'Postgres version to use for testing (e. g. v17)'
+        default: 'v17'
```

Comment:
Should it accept a JSON array as well?

Reply:
This input has been modified to accept JSON.